.github/ISSUE_TEMPLATE/feature-request-or-enhancement---.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
this change was already moved over I believe https://github.com/carbon-design-system/carbon/blob/master/.github/ISSUE_TEMPLATE/feature-request-or-enhancement.md

Reply:
Good point, I think recent merge in this PR subsumed this.